### PR TITLE
Replace torch.tensor by my_tensor.clone().detach()

### PR DESCRIPTION
**Pull request by Louis-Dupont**
PyTorch raises a warning when using torch.tensor instead of my_tensor.clone().detach().

### CODE REVIEW

**Review by ofrimasad — APPROVED** [review]
LGTM
